### PR TITLE
[Backport] Fix swatch overflow issue

### PR DESCRIPTION
**Pull request by JamieRuroc**
### Description (*)

   When dealing with multiple store views with different languages the swatch input fields become too small to use and they move off the screen with no way to get to them. This fix adds a min-width to the input fields and handles swatch overflow with a scrollbar.

### Manual testing scenarios (*)

1. Add multiple store views and languages to your store.
2. Open an attribute page within the Backend (Stores > Product > "Attribute").
3. Add more store views / languages until overflow happens.

Backport for: https://github.com/magento/magento2/pull/19932

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @JamieRuroc. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.2-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=19933) <br/>All committers have signed the CLA.

File: app/code/Magento/Swatches/view/adminhtml/web/css/swatches.css
Comment (sidolov) [inline app/code/Magento/Swatches/view/adminhtml/web/css/swatches.css]: Please, use correct indentation for styles

**Review by sidolov — CHANGES_REQUESTED** [review]

File: app/code/Magento/Swatches/view/adminhtml/web/css/swatches.css
Comment (JamieRuroc) [reply app/code/Magento/Swatches/view/adminhtml/web/css/swatches.css]: Indentation has been corrected.

**Comment by JamieRuroc** [comment]
@sidolov this version has also been updated, thank you.

**Comment by sidolov** [comment]
Hi @JamieRuroc , thank you for the update! We have the rule that all changes should be delivered to the 2.3-develop first, so, I added "on hold" label for this pull request right now and I'll remove it once #19932 will be merged.
Thank you!

**Comment by sidolov** [comment]
Hi @JamieRuroc , I am closing this PR because original was closed due to inactivity. 
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**
